3rdparty/nvtt/nvcore/nvcore.h

```diff
@@ -1,4 +1,4 @@
-// This code is in the public domain -- Ignacio Casta�o <castano@gmail.com>
+// This code is in the public domain -- Ignacio Casta�o <castano@gmail.com>
```

Comment:
Undo this line because the ñ in Ignacio Castaño's name was messed up (presumably a mistake by your text editor)

Reply:
Hmm, that's interesting. I'm actually using VSCode to edit the files and it should support UTF-8 just fine.